components/builder-web/opensearch.xml

```diff
@@ -0,0 +1,11 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
+                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+  <ShortName>habitat.sh</ShortName>
```

Comment:
Overall this looks great, thanks for adding it. Would you mind changing the ShortName and Description to:
ShortName: Habitat Search
Description: Use bldr.habitat.sh to search for packages on Habitat Builder.

Thanks!

Reply:
Will do, np!

Reply:
@ryankeairns done!

Reply:
Thank you!